cmd/cli/plugin/secret/main.go

```diff
@@ -10,10 +10,10 @@ import (
 	_ "k8s.io/client-go/plugin/pkg/client/auth"
 
 	cliv1alpha1 "github.com/vmware-tanzu/tanzu-framework/apis/cli/v1alpha1"
+	"github.com/vmware-tanzu/tanzu-framework/cli/runtime/buildinfo"
```

Comment:
Same comment as above, you should not change any other plugins files. We can keep using ```buildinfo``` as it is for now